velox/exec/HashBuild.cpp

```diff
@@ -700,7 +701,7 @@ bool HashBuild::finishHashBuild() {
       VELOX_CHECK(build);
       if (build->joinHasNullKeys_) {
         joinHasNullKeys_ = true;
-        if (isNullAwareAntiJoin(joinType_)) {
```

Comment:
nit: Do we want to keep these utilities(node instead of join type) like as anonymous function in this cc file or PlanNode.h?

Reply:
I'm not sure about this just yet. I think it might be confusing that some of these take joinType and some joinNode. I'm also hoping for sharing logic between null-aware branches of different join types.